etc/default/default_config.yaml

```diff
@@ -25,33 +31,6 @@ data_ingestion:
     - name: remove_ligature_st
       pattern: 'ﬆ'
       replace_with: 'st'
-    - name: replace_fi
```

Comment:
If these are no longer necessary, they should also be removed in the .j2 template